simplified-ui-accounts/src/main/res/layout/account.xml

```diff
@@ -258,11 +258,11 @@
     </androidx.constraintlayout.widget.ConstraintLayout>
 
     <androidx.constraintlayout.widget.ConstraintLayout
-      android:visibility="gone"
       android:id="@+id/accountReportIssue"
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
-      android:background="?attr/selectableItemBackground">
+      android:background="?attr/selectableItemBackground"
+      android:visibility="visible">
```

Comment:
Visible is the default so you can just remove this attribute entirely.

Reply:
Oops! :+1:

Reply:
Sorry, total nitpick 😅